monai/data/box_utils.py

```diff
@@ -572,3 +572,261 @@ def convert_box_to_standard_mode(
             box_convert_mode(boxes=boxes, src_mode="xxyyzz", dst_mode="xyzxyz")
     """
     return convert_box_mode(boxes=boxes, src_mode=mode, dst_mode=StandardMode())
+
+
+def check_boxes(boxes: NdarrayOrTensor) -> bool:
+    """
+    This function checks whether the box size is non-negative.
+
+    Args:
+        boxes: bounding box, Nx4 or Nx6 torch tensor or ndarray. The box mode is assumed to be ``StandardMode``
```

Comment:
Maybe change all the `bounding box` of doc-string to `bounding boxes` in this PR, to be consistent with:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/box_utils.py#L502

Thanks.